# Comparing `tmp/nanoprep_ffm-0.0.5-py3-none-any.whl.zip` & `tmp/nanoprep_ffm-0.0.6-py3-none-any.whl.zip`

## zipinfo {}

```diff
@@ -1,22 +1,22 @@
-Zip file size: 15299 bytes, number of entries: 20
+Zip file size: 16187 bytes, number of entries: 20
 -rw-rw-r--  2.0 unx        0 b- defN 23-Apr-06 14:17 NanoPreP/__init__.py
--rw-rw-r--  2.0 unx     5469 b- defN 23-Apr-07 09:43 NanoPreP/__main__.py
+-rw-rw-r--  2.0 unx     5425 b- defN 23-Apr-08 06:34 NanoPreP/__main__.py
 -rw-rw-r--  2.0 unx        0 b- defN 23-Apr-06 14:17 NanoPreP/aligntools/__init__.py
--rw-rw-r--  2.0 unx     1881 b- defN 23-Apr-07 10:06 NanoPreP/aligntools/edlibAligner.py
+-rw-rw-r--  2.0 unx     2824 b- defN 23-Apr-08 08:29 NanoPreP/aligntools/edlibAligner.py
 -rw-rw-r--  2.0 unx        0 b- defN 23-Apr-06 14:17 NanoPreP/paramtools/__init__.py
 -rw-rw-r--  2.0 unx     3802 b- defN 23-Apr-06 14:17 NanoPreP/paramtools/argParser.py
 -rw-rw-r--  2.0 unx     1592 b- defN 23-Apr-07 06:07 NanoPreP/paramtools/paramsets.py
--rw-rw-r--  2.0 unx     4678 b- defN 23-Apr-07 10:09 NanoPreP/preptools/Annotator.py
+-rw-rw-r--  2.0 unx     9302 b- defN 23-Apr-14 07:32 NanoPreP/preptools/Annotator.py
 -rw-rw-r--  2.0 unx     1287 b- defN 23-Apr-06 14:17 NanoPreP/preptools/Processor.py
 -rw-rw-r--  2.0 unx        0 b- defN 23-Apr-06 14:17 NanoPreP/preptools/__init__.py
 -rw-rw-r--  2.0 unx     1332 b- defN 23-Apr-06 14:17 NanoPreP/preptools/polyFinder.py
 -rw-rw-r--  2.0 unx      545 b- defN 23-Apr-07 05:16 NanoPreP/seqtools/FastqIO.py
 -rw-rw-r--  2.0 unx     6094 b- defN 23-Apr-06 14:17 NanoPreP/seqtools/SeqFastq.py
 -rw-rw-r--  2.0 unx        0 b- defN 23-Apr-06 14:17 NanoPreP/seqtools/__init__.py
--rwxrwxr-x  2.0 unx     1068 b- defN 23-Apr-07 10:16 nanoprep_ffm-0.0.5.dist-info/LICENSE
--rw-rw-r--  2.0 unx    10637 b- defN 23-Apr-07 10:16 nanoprep_ffm-0.0.5.dist-info/METADATA
--rw-rw-r--  2.0 unx       92 b- defN 23-Apr-07 10:16 nanoprep_ffm-0.0.5.dist-info/WHEEL
--rw-rw-r--  2.0 unx       52 b- defN 23-Apr-07 10:16 nanoprep_ffm-0.0.5.dist-info/entry_points.txt
--rw-rw-r--  2.0 unx        9 b- defN 23-Apr-07 10:16 nanoprep_ffm-0.0.5.dist-info/top_level.txt
--rw-rw-r--  2.0 unx     1704 b- defN 23-Apr-07 10:16 nanoprep_ffm-0.0.5.dist-info/RECORD
-20 files, 40242 bytes uncompressed, 12477 bytes compressed:  69.0%
+-rwxrwxr-x  2.0 unx     1068 b- defN 23-Apr-18 09:08 nanoprep_ffm-0.0.6.dist-info/LICENSE
+-rw-rw-r--  2.0 unx    10637 b- defN 23-Apr-18 09:08 nanoprep_ffm-0.0.6.dist-info/METADATA
+-rw-rw-r--  2.0 unx       92 b- defN 23-Apr-18 09:08 nanoprep_ffm-0.0.6.dist-info/WHEEL
+-rw-rw-r--  2.0 unx      104 b- defN 23-Apr-18 09:08 nanoprep_ffm-0.0.6.dist-info/entry_points.txt
+-rw-rw-r--  2.0 unx        9 b- defN 23-Apr-18 09:08 nanoprep_ffm-0.0.6.dist-info/top_level.txt
+-rw-rw-r--  2.0 unx     1705 b- defN 23-Apr-18 09:08 nanoprep_ffm-0.0.6.dist-info/RECORD
+20 files, 45818 bytes uncompressed, 13365 bytes compressed:  70.8%
```

## zipnote {}

```diff
@@ -36,26 +36,26 @@
 
 Filename: NanoPreP/seqtools/SeqFastq.py
 Comment: 
 
 Filename: NanoPreP/seqtools/__init__.py
 Comment: 
 
-Filename: nanoprep_ffm-0.0.5.dist-info/LICENSE
+Filename: nanoprep_ffm-0.0.6.dist-info/LICENSE
 Comment: 
 
-Filename: nanoprep_ffm-0.0.5.dist-info/METADATA
+Filename: nanoprep_ffm-0.0.6.dist-info/METADATA
 Comment: 
 
-Filename: nanoprep_ffm-0.0.5.dist-info/WHEEL
+Filename: nanoprep_ffm-0.0.6.dist-info/WHEEL
 Comment: 
 
-Filename: nanoprep_ffm-0.0.5.dist-info/entry_points.txt
+Filename: nanoprep_ffm-0.0.6.dist-info/entry_points.txt
 Comment: 
 
-Filename: nanoprep_ffm-0.0.5.dist-info/top_level.txt
+Filename: nanoprep_ffm-0.0.6.dist-info/top_level.txt
 Comment: 
 
-Filename: nanoprep_ffm-0.0.5.dist-info/RECORD
+Filename: nanoprep_ffm-0.0.6.dist-info/RECORD
 Comment: 
 
 Zip file comment:
```

## NanoPreP/__main__.py

```diff
@@ -1,17 +1,15 @@
-#!usr/bin/env Python
-from pathlib import Path
-import re
 from NanoPreP.preptools.Annotator import Annotator
 from NanoPreP.preptools.Processor import Processor
 from NanoPreP.seqtools.FastqIO import FastqIO
 from NanoPreP.seqtools.SeqFastq import SeqFastq
 from NanoPreP.paramtools.paramsets import Params, Defaults
 from NanoPreP.paramtools.argParser import parser
 from datetime import datetime
+from pathlib import Path
 import sys
 import json
 import gzip
 
 def main():
     # parse arguments
     args = parser.parse_args()
@@ -28,15 +26,14 @@
             opts = ", ".join([i.__repr__() for i in Params.keys()])
             raise Exception(msg + opts)
 
     # get parameter from config
     if args.config:
         config = json.load(open(args.config))
         params.update(config)
-        pass
 
     # get command line arguments from ArgumentParser
     for k, v in vars(args).items():
         if not v == Defaults[k]:
             params[k] = v    
         elif f"--{k}" in sys.argv:
             params[k] = v
```

## NanoPreP/aligntools/edlibAligner.py

```diff
@@ -1,27 +1,28 @@
 """Caller of edlib.align()"""
-from typing import Tuple
+from typing import Tuple, List
 import edlib
 
 class edlibAligner:
     def singleAlign(
         query: str,
         target: str,
         mode: str,
         task: str,
         pid: float,
         tie_breaking: str = "middle"
     ) -> dict:
         # call edlib for the alignment
+        k = -1 if pid == -1 else round((1 - pid) * len(query))
         res = edlib.align(
             query,
             target,
             mode,
             task,
-            round((1 - pid) * len(query))
+            k
         )
 
         # add 2 fields: `pid` and `location` to `res`
         if res["editDistance"] >= 0:
             # pid
             res["pid"] = 1 - res["editDistance"] / len(query)
             
@@ -66,7 +67,40 @@
 
             # if the new alignment result is better
             if new["pid"] > res["pid"]:
                 res = new
                 name = qname
 
         return name, res
+
+
+    def ntopAligns(
+            query: str,
+            target: str,
+            mode: str,
+            task: str,
+            pid: float,
+            n: int,
+            tie_breaking: str = "middle"
+        ) -> List[dict]:
+        out = []
+        lastloc = None
+        for _ in range(n):
+            # mask target sequence
+            if lastloc:
+                target = target[:lastloc[0]] + \
+                    "N" * (lastloc[1] - lastloc[0]) + \
+                    target[lastloc[1]:]
+            
+            # align
+            res =  edlibAligner.singleAlign(
+                    query,
+                    target,
+                    mode,
+                    task,
+                    pid,
+                    tie_breaking
+                )
+            lastloc = res["location"] if res["pid"] != -1 else None
+            out.append(res)            
+                
+        return out
```

## NanoPreP/preptools/Annotator.py

```diff
@@ -1,10 +1,12 @@
 from NanoPreP.seqtools.SeqFastq import SeqFastq, SeqAnnot
 from NanoPreP.aligntools.edlibAligner import edlibAligner as aligner
 from NanoPreP.preptools.polyFinder import polyFinder
+from collections import namedtuple
+from typing import Tuple
 import re
 
 class Annotator(object):
     def __init__(
         self,
         p5_sense: str,
         p3_sense: str,
@@ -55,15 +57,15 @@
         self.pid_body = pid_body
         self.w = w  # window size for polymer trimming
         self.k = k  # required number of polymers in the window
         self.poly = polymers
         return
 
     # annotate features on SeqFastq
-    def annotate(self, read: SeqFastq) -> None:
+    def annotate(self, read: SeqFastq, always: bool = False) -> None:
         # reset annot
         read.annot = SeqAnnot()
 
         # detect fusion
         if len(read) > \
                 (self.isl5[1] - self.isl5[0] + self.isl3[1] - self.isl3[0]):
             name, res = aligner.bestAlign(
@@ -76,15 +78,16 @@
                 read.seq[self.isl5[1]:self.isl3[0]],
                 mode="HW",
                 task="locations",
                 pid=self.pid_body
             )
             if res["pid"] > 0:
                 read.annot.fusion = 1
-                return
+                if not always:
+                    return
 
         # align 5' primers to 5' isl
         strand, res = aligner.bestAlign(
             {1: self.p5_sense, -1: self.p5_anti},
             read.seq[self.isl5[0]:self.isl5[1]],
             mode="HW",
             task="locations",
@@ -126,9 +129,166 @@
                 if strand * read.annot.strand > 0:
                     # get finder
                     finder = {
                         5: polyFinder.lfind,
                         3: polyFinder.rfind
                     }[end]
                     finder(read, n, max_n, k=self.k, w=self.w)
-
         return
+
+    
+class Optimizer:
+    def __init__(
+            self,
+            p5_sense: str,
+            p3_sense: str,
+            p5_anti: str = None,
+            p3_anti: str = None,
+            isl5: tuple = None,
+            isl3: tuple = None
+        ) -> None:
+        self.p5_sense = p5_sense
+        self.p3_sense = p3_sense
+        self.p5_anti = p5_anti if p5_anti else SeqFastq.reverse_complement_static(p3_sense)
+        self.p3_anti = p3_anti if p3_anti else SeqFastq.reverse_complement_static(p5_sense)
+        self.isl5 = isl5
+        self.isl3 = isl3
+        return 
+    
+    def alignPNbests(self, read: SeqFastq, plen: int, nbests: int) -> dict[str,namedtuple]:
+        """_summary_
+
+        Args:
+            read (SeqFastq): _description_
+            plen (int): _description_
+            nbests (int): _description_
+
+        Returns:
+            dict: {"primer_name": SimpleResult(pid, location)}
+        """
+        SimpleResult = namedtuple("SimpleResult", "pid, location")
+        out = {}
+        out["p5_sense"] = \
+            [SimpleResult(i["pid"], i["location"][1]) for i in 
+            aligner.ntopAligns(
+                self.p5_sense[-plen:],
+                read.seq,
+                "HW",
+                "locations",
+                -1,
+                nbests
+            )
+        ]
+        out["p5_anti"] = \
+            [SimpleResult(i["pid"], i["location"][1]) for i in 
+            aligner.ntopAligns(
+                self.p5_anti[-plen:],
+                read.seq,
+                "HW",
+                "locations",
+                -1,
+                nbests
+            )
+        ]
+        out["p3_sense"] =  \
+            [SimpleResult(i["pid"], len(read.seq) - i["location"][0]) for i in 
+            aligner.ntopAligns(
+                self.p3_sense[:plen],
+                read.seq,
+                "HW",
+                "locations",
+                -1,
+                nbests
+            )
+        ]
+        out["p3_anti"] = \
+            [SimpleResult(i["pid"], len(read.seq) - i["location"][0]) for i in 
+            aligner.ntopAligns(
+                self.p3_anti[:plen],
+                read.seq,
+                "HW",
+                "locations",
+                -1,
+                nbests
+            )
+        ]
+        return out
+    
+    
+    def alignP2P(self, plen: int) -> dict:
+        """Align primer to primers (5' to 5'; 3' to 3')
+
+        Args:
+            plen (int): the length of the primer substring to be aligned
+
+        Returns:
+            dict: {"p5": pid, "p3": pid}
+        """
+        out = {}
+        out["p5"] = aligner.singleAlign(
+                self.p5_sense[-plen:],
+                self.p5_anti[-plen:],
+                "HW",
+                "locations",
+                -1
+            )["pid"]
+        out["p3"] = aligner.singleAlign(
+                self.p3_sense[:plen],
+                self.p3_anti[:plen],
+                "HW",
+                "locations",
+                -1
+            )["pid"]
+    
+        return out
+    
+    
+    def alignInB(self, read: SeqFastq) -> dict[str, Tuple[float, float]]:
+        """Align primer to ISL and read body
+
+        Args:
+            read (SeqFastq): _description_
+
+        Returns:
+            Dict[Tuple[float, float]]: 
+            {
+                "p5": (pid_isl, pid_body),
+                "p3": (pid_isl, pid_body)
+            }
+        """
+        
+        out = {}
+        # align 5' primers to 5' isl
+        strand, res = aligner.bestAlign(
+            {1: self.p5_sense, -1: self.p5_anti},
+            read.seq[self.isl5[0]:self.isl5[1]],
+            mode="HW",
+            task="locations",
+            pid=-1
+        )
+        pid_body = aligner.singleAlign(
+            {1: self.p5_sense, -1: self.p5_anti}[strand],
+            read.seq[self.isl5[1]:self.isl3[0]],
+            mode="HW",
+            task="locations",
+            pid=-1
+        )["pid"]
+        out["p5"] = (res["pid"], pid_body)
+
+        # align 3' primers to 3' isl
+        strand, res = aligner.bestAlign(
+            {1: self.p3_sense, -1: self.p3_anti},
+            read.seq[self.isl3[0]:self.isl3[1]],
+            mode="HW",
+            task="locations",
+            pid=-1,
+        )
+        pid_body = aligner.singleAlign(
+            {1: self.p3_sense, -1: self.p3_anti}[strand],
+            read.seq[self.isl5[1]:self.isl3[0]],
+            mode="HW",
+            task="locations",
+            pid=-1
+        )["pid"]
+        out["p3"] = (res["pid"], pid_body)
+        
+        return out
```

## Comparing `nanoprep_ffm-0.0.5.dist-info/LICENSE` & `nanoprep_ffm-0.0.6.dist-info/LICENSE`

 * *Files identical despite different names*

## Comparing `nanoprep_ffm-0.0.5.dist-info/METADATA` & `nanoprep_ffm-0.0.6.dist-info/METADATA`

 * *Files 0% similar despite different names*

```diff
@@ -1,10 +1,10 @@
 Metadata-Version: 2.1
 Name: nanoprep-ffm
-Version: 0.0.5
+Version: 0.0.6
 Summary: A fully-equipped, fast, and memory-efficient pre-processor for ONT transcriptomic data
 Home-page: https://github.com/Woodformation1136/NanoPreP
 Author: Chia-Chen Chu
 Author-email: jerry955071@gmail.com
 Description-Content-Type: text/markdown
 License-File: LICENSE
 Requires-Dist: edlib (>=1.3.8)
```

## Comparing `nanoprep_ffm-0.0.5.dist-info/RECORD` & `nanoprep_ffm-0.0.6.dist-info/RECORD`

 * *Files 20% similar despite different names*

```diff
@@ -1,20 +1,20 @@
 NanoPreP/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
-NanoPreP/__main__.py,sha256=xgfeLAi8ny-Ywzih32StBJGTngDRKC7GH7XW-dbW97k,5469
+NanoPreP/__main__.py,sha256=RhT1hrJaHgbfhGCjLuv2vc-5aUa3_DMZDZtrudgenPE,5425
 NanoPreP/aligntools/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
-NanoPreP/aligntools/edlibAligner.py,sha256=vDPDpcli6uAYsR2Xuqnd5FzKKcgRvvSxt_WnEtPBEvU,1881
+NanoPreP/aligntools/edlibAligner.py,sha256=mTI10q_BOwgBrgNONybbrVwpzS7qE4P_tWcCMBw2fHE,2824
 NanoPreP/paramtools/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
 NanoPreP/paramtools/argParser.py,sha256=07UdB5ee3sl5AgFiej4stEVf1Ps-bmq0-IMjpTf7LJI,3802
 NanoPreP/paramtools/paramsets.py,sha256=c_x-5tUT9j9A74gtNe331myiPnPBSf0eEteAmlh_6EI,1592
-NanoPreP/preptools/Annotator.py,sha256=XRUBNlcAFDGS-ZlKeG7DqBDw__xuPs3iufFJAisHcQc,4678
+NanoPreP/preptools/Annotator.py,sha256=lEILFK0KE7V8YdJ7rdOH3KVCuIrUeiQdP043Z5ZyuZA,9302
 NanoPreP/preptools/Processor.py,sha256=oMIhPsUN2f9VLSnMBimfeA0OLSDunlJm2leIX9GIYGc,1287
 NanoPreP/preptools/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
 NanoPreP/preptools/polyFinder.py,sha256=wNvsvpmiItRdNofCWIf5w5qwqFdx2cdK56vd-YfhkCU,1332
 NanoPreP/seqtools/FastqIO.py,sha256=1_OQvF88sdZBjNZ7_hF3U-hlitGTjn9bmjklnVFZymc,545
 NanoPreP/seqtools/SeqFastq.py,sha256=DJwRhEbqqBfw12X3yX2lwvvSNzNL_r1RBB4rLXzgoCc,6094
 NanoPreP/seqtools/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
-nanoprep_ffm-0.0.5.dist-info/LICENSE,sha256=uVVSzhIqL8JFWzS-neMC5ERsJzqrtTSxWLnaMxDDN_s,1068
-nanoprep_ffm-0.0.5.dist-info/METADATA,sha256=IpALR8r2YuRD_Mqh9VL9wv2RmkATyF1Q3AmGMgmJSXc,10637
-nanoprep_ffm-0.0.5.dist-info/WHEEL,sha256=pkctZYzUS4AYVn6dJ-7367OJZivF2e8RA9b_ZBjif18,92
-nanoprep_ffm-0.0.5.dist-info/entry_points.txt,sha256=oBZzSPYTN0uOm1U6ePwc5aL79kfBoVHmpss_5AAt1xM,52
-nanoprep_ffm-0.0.5.dist-info/top_level.txt,sha256=w-4ylV_liD0-xevxfLKasE0p1jea4AXdPAwlcE4oc3U,9
-nanoprep_ffm-0.0.5.dist-info/RECORD,,
+nanoprep_ffm-0.0.6.dist-info/LICENSE,sha256=uVVSzhIqL8JFWzS-neMC5ERsJzqrtTSxWLnaMxDDN_s,1068
+nanoprep_ffm-0.0.6.dist-info/METADATA,sha256=UDk7iQm5_J-L-wKdp8IZHEc-57nZZVeBuYjbCfZK4DA,10637
+nanoprep_ffm-0.0.6.dist-info/WHEEL,sha256=pkctZYzUS4AYVn6dJ-7367OJZivF2e8RA9b_ZBjif18,92
+nanoprep_ffm-0.0.6.dist-info/entry_points.txt,sha256=29o3qAaEXsokaGdmB3KbEE1wILOgGlLzt80Wlz1XR5c,104
+nanoprep_ffm-0.0.6.dist-info/top_level.txt,sha256=w-4ylV_liD0-xevxfLKasE0p1jea4AXdPAwlcE4oc3U,9
+nanoprep_ffm-0.0.6.dist-info/RECORD,,
```

